sys/fscontrol.c

```diff
@@ -614,14 +624,18 @@ NTSTATUS DokanMountVolume(__in PDEVICE_OBJECT DiskDevice, __in PIRP Irp) {
     RtlCopyMemory(dokanControl.UNCName, dcb->UNCName->Buffer,
                   dcb->UNCName->Length);
   }
-  mountEntry = FindMountEntry(dcb->Global, &dokanControl);
-  if (mountEntry != NULL) {
+  mountEntry = FindMountEntry(dcb->Global, &dokanControl, TRUE);
+  if (mountEntry != NULL && &mountEntry->MountControl) {
```

Comment:
If the MountControl pointer is still valid. Depending on timing it could be null if at the same time somebody removed the mounEntry.